region.go

```diff
@@ -44,115 +44,149 @@ func GetRegion(zone string) (region string, err error) {
+		Description:  "Madrid, Spain",
+		VPCRegion:    "eu-es",
+		COSRegion:    "eu-de",		// @HACK - PowerVS says COS not supported in this region
 		Zones: []string{
 			"mad02",
 			"mad04",
 		},
-		SysTypes:    []string{"s1022"},
+		SysTypes:     []string{"s1022"},
+		IPISupported: true,
 	},
 	"mon": {
-		Description: "Montreal, Canada",
-		VPCRegion:   "ca-tor",
-		COSRegion:   "ca-tor",
-		Zones:       []string{"mon01"},
-		SysTypes:    []string{"s922", "e980"},
+		Description:  "Montreal, Canada",
+		VPCRegion:    "ca-tor",
+		COSRegion:    "ca-tor",
+		Zones:        []string{"mon01"},
+		SysTypes:     []string{"s922", "e980"},
+		IPISupported: false,
 	},
 	"osa": {
-		Description: "Osaka, Japan",
-		VPCRegion:   "jp-osa",
-		COSRegion:   "jp-osa",
-		Zones:       []string{"osa21"},
-		SysTypes:    []string{"s922", "e980"},
+		Description:  "Osaka, Japan",
+		VPCRegion:    "jp-osa",
+		COSRegion:    "jp-osa",
+		Zones:        []string{"osa21"},
+		SysTypes:     []string{"s922", "e980"},
+		IPISupported: false,
 	},
 	"syd": {
-		Description: "Sydney, Australia",
-		VPCRegion:   "au-syd",
-		COSRegion:   "au-syd",
+		Description:  "Sydney, Australia",
+		VPCRegion:    "au-syd",
+		COSRegion:    "au-syd",
 		Zones: []string{
 			"syd04",
 			"syd05",
 		},
-		SysTypes:    []string{"s922", "e980"},
+		SysTypes:     []string{"s922", "e980"},
+		IPISupported: false,
 	},
 	"sao": {
-		Description: "São Paulo, Brazil",
-		VPCRegion:   "br-sao",
-		COSRegion:   "br-sao",
-		Zones:       []string{
+		Description:  "São Paulo, Brazil",
+		VPCRegion:    "br-sao",
+		COSRegion:    "br-sao",
+		Zones:        []string{
 			"sao01",
 			"sao04",
 		},
-		SysTypes:    []string{"s922", "e980"},
+		SysTypes:     []string{"s922", "e980"},
+		IPISupported: true,
 	},
 	"tok": {
-		Description: "Tokyo, Japan",
-		VPCRegion:   "jp-tok",
-		COSRegion:   "jp-tok",
-		Zones:       []string{"tok04"},
-		SysTypes:    []string{"s922", "e980"},
+		Description:  "Tokyo, Japan",
+		VPCRegion:    "jp-tok",
+		COSRegion:    "jp-tok",
+		Zones:        []string{"tok04"},
+		SysTypes:     []string{"s922", "e980"},
+		IPISupported: false,
 	},
 	"us-east": {
-		Description: "Washington DC, USA",
-		VPCRegion:   "us-east",
-		COSRegion:   "us-east",
-		Zones:       []string{"us-east"},
-		SysTypes:    []string{}, // Missing
+		Description:  "Washington DC, USA",
+		VPCRegion:    "us-east",
+		COSRegion:    "us-east",
+		Zones:        []string{"us-east"},
+		SysTypes:     []string{}, // Missing
+		IPISupported: false,
 	},
 	"wdc": {
-		Description: "Washington DC, USA",
-		VPCRegion:   "us-east",
-		COSRegion:   "us-east",
+		Description:  "Washington DC, USA",
+		VPCRegion:    "us-east",
+		COSRegion:    "us-east",
 		Zones: []string{
 			"wdc06",
 			"wdc07",
 		},
-		SysTypes:    []string{"s922", "e980"},
+		SysTypes:     []string{"s922", "e980"},
+		IPISupported: true,
 	},
 }
 
+var IPIRegions = map[string]Region{}
```

Comment:
It seems unused

Reply:
It is used.  Have you searched the code?